code/mob/dead.dm

```diff
@@ -8,6 +8,7 @@
 /mob/dead/New()
 	..()
 	src.flags |= UNCRUSHABLE
+	APPLY_ATOM_PROPERTY(src, PROP_ATOM_FLOATING, src)
```

Comment:
should probably apply to `/mob/living/intangible` also

Reply:
the old one didn't check! but will see if that causes problems